composer.json

```diff
@@ -27,8 +27,8 @@
     },
     "require-dev": {
         "atoum/atoum": "^2.8 || ^3.0",
-        "m6web/coke": "~1.2",
-        "m6web/symfony2-coding-standard": "~1.2",
+        "m6web/coke": "^2.0",
+        "m6web/symfony2-coding-standard": "^3.0",
```

Comment:
Those two packages are now deprecated.

Now we use `php-cs-fixer` to check coding style.
So you should use [php-cs-fixer-config](https://github.com/BedrockStreaming/php-cs-fixer-config) instead.

Reply:
I expected that